### PR TITLE
Sass should be capital letter

### PR DESCRIPTION
**Pull request by cht8687**
:smile:

### CODE REVIEW

**Comment by pixcai** [comment]
I think it makes no sense to discuss this question...

**Comment by cht8687** [comment]
@pixcai why? 

https://en.wikipedia.org/wiki/Sass_(stylesheet_language)

http://sass-lang.com/

I can't find anywhere which has `sass` instead of `Sass`, please correct me if I am wrong.

File: README.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ Be sure you have added `node-modules` to your sass's [load_paths](http://stackov
 
 ## Contributing
 
-Before contributing, please ensure you have `sass` and `scss-lint` installed. You can install them by running this:
+Before contributing, please ensure you have `Sass` and `scss-lint` installed. You can install them by running this:
```
Comment (xcatliu) [inline README.md]: Here, `sass` means a cli, should keep use `sass`, not `Sass`

**Comment by pixcai** [comment]
After all, this is not papers. Everyone knows what `sass` means, that's enough!

File: README.md (hunk 1 above)
Comment (cht8687) [reply README.md]: thanks, I will update this.

**Comment by cht8687** [comment]
@pixcai  Yes, this is not paper, but this is open-source. Everyone can find things to improve. right? we need make everything perfect, shouldn't we? I think in terms of CSS, even a tiny 1 px difference will make difference. Do you agree with me? :smile:  thanks @xcatliu  to make this awesome project. I will definitely learn more about the code part and try to contribute.

**Pull request merged**